### DISCUSSION TITLE
I don't have the "show pcap for this event" button

### DISCUSSION THREAD

**Discussion by jackface0** [Security-Onion-Solutions/securityonion#6758] [Unsupported Versions] [answered]
Hi All,

My security onion version is 2.3.90.

I'm trying to investigate an alert:
GPL NETBIOS DCERPC Remote Activation bind attempt (Attempted Administrator Privilege Gain)

In the training youtube video I watched they had a "show pcap for this event" link when drilling down on the event but it's just not there for me. Is there something I need to enable to get it or am I looking at something different ?

I found this link www.microsoft.com/technet/security/bulletin/MS03-039.mspx in the event details but the page does not seem to exist anymore but I think its related to: https://docs.microsoft.com/en-us/security-updates/securitybulletins/2003/ms03-039

The rule reference lands on a general looking page too
https://doc.emergingthreats.net/2102251

I'm trying to learn about this rule/alert to see if its false positive or could be something bad. The source IP 10.4.8.1 is a firewall looks like its contacting various LAN hosts on TCP port 135. I'm not sure at the moment if this is normal behaviour for the firewall, maybe its trying to do user ID or something.

Full event:
`
{"@timestamp":"2022-01-05T04:59:37.301Z","@version":"1","destination.ip":"10.4.9.16","destination.port":135,"ecs.version":"1.11.0","event.category":"network","event.dataset":"alert","event.module":"suricata","event.severity":3,"event.severity_label":"high","host.name":"securityonion","ingest.timestamp":"2022-01-05T04:59:37.998Z","log.file.path":"/nsm/suricata/eve-2022-01-05-04:07.json","log.id.uid":"394242123468891","log.offset":11309206,"message":"{\"timestamp\":\"2022-01-05T04:59:37.301175+0000\",\"flow_id\":394242123468891,\"in_iface\":\"bond0\",\"event_type\":\"alert\",\"src_ip\":\"10.4.8.1\",\"src_port\":53736,\"dest_ip\":\"10.4.9.16\",\"dest_port\":135,\"proto\":\"TCP\",\"metadata\":{\"flowbits\":[\"dce.isystemactivator.bind\"]},\"community_id\":\"1:tvddB9xoz8iEuC+BWH5oPBK+D/M=\",\"alert\":{\"action\":\"allowed\",\"gid\":1,\"signature_id\":2102251,\"rev\":16,\"signature\":\"GPL NETBIOS DCERPC Remote Activation bind attempt\",\"category\":\"Attempted Administrator Privilege Gain\",\"severity\":1,\"metadata\":{\"created_at\":[\"2010_09_23\"],\"updated_at\":[\"2010_09_23\"]},\"rule\":\"alert tcp $EXTERNAL_NET any -> $HOME_NET 135 (msg:\\\"GPL NETBIOS DCERPC Remote Activation bind attempt\\\"; flow:to_server,established; content:\\\"|05|\\\"; content:\\\"|0B|\\\"; within:1; distance:1; byte_test:1,&,1,0,relative; content:\\\"|B8|J|9F|M|1C|}|CF 11 86 1E 00| |AF|n|7C|W\\\"; within:16; distance:29; tag:session,5,packets; reference:bugtraq,8234; reference:bugtraq,8458; reference:cve,2003-0528; reference:cve,2003-0605; reference:cve,2003-0715; reference:nessus,11798; reference:nessus,11835; reference:url,www.microsoft.com/technet/security/bulletin/MS03-039.mspx; classtype:attempted-admin; sid:2102251; rev:16; metadata:created_at 2010_09_23, updated_at 2010_09_23;)\"},\"app_proto\":\"dcerpc\",\"payload_printable\":\"..........R......................J.M.}..... .n|W.....]..........+.H`........gE.k`P..+......F0D..0..\\n+.....7..\\n.2.0NTLMSSP........`.... .......)...WORKGROUPPA-3220\",\"stream\":1,\"packet\":\"AAAMB6wItAwl4EBACABFAAA0u+pAAEAGWcEKBAgBCgQJENHoAIcWUVojZPDng4AQAHukAwAAAQEIChl34mgJ/hfw\",\"packet_info\":{\"linktype\":1}}","metadata.beat":"filebeat","metadata.ip_address":"172.17.0.1","metadata.type":"_doc","metadata.version":"7.15.2","network.community_id":"1:tvddB9xoz8iEuC+BWH5oPBK+D/M=","network.data.decoded":"..........R......................J.M.}..... .n|W.....]..........+.H`........gE.k`P..+......F0D..0..\n+.....7..\n.2.0NTLMSSP........`.... .......)...WORKGROUPPA-3220","network.transport":"TCP","observer.name":"securityonion","rule.action":"allowed","rule.category":"Attempted Administrator Privilege Gain","rule.gid":1,"rule.metadata.created_at":["2010_09_23"],"rule.metadata.updated_at":["2010_09_23"],"rule.name":"GPL NETBIOS DCERPC Remote Activation bind attempt","rule.reference":"https://doc.emergingthreats.net/2102251","rule.rev":16,"rule.rule":"alert tcp $EXTERNAL_NET any -> $HOME_NET 135 (msg:\"GPL NETBIOS DCERPC Remote Activation bind attempt\"; flow:to_server,established; content:\"|05|\"; content:\"|0B|\"; within:1; distance:1; byte_test:1,&,1,0,relative; content:\"|B8|J|9F|M|1C|}|CF 11 86 1E 00| |AF|n|7C|W\"; within:16; distance:29; tag:session,5,packets; reference:bugtraq,8234; reference:bugtraq,8458; reference:cve,2003-0528; reference:cve,2003-0605; reference:cve,2003-0715; reference:nessus,11798; reference:nessus,11835; reference:url,www.microsoft.com/technet/security/bulletin/MS03-039.mspx; classtype:attempted-admin; sid:2102251; rev:16; metadata:created_at 2010_09_23, updated_at 2010_09_23;)","rule.ruleset":"Snort GPL","rule.severity":1,"rule.uuid":"2102251","source.ip":"10.4.8.1","source.port":53736,"tags":["beats_input_codec_plain_applied"],"soc_id":"R7-bKH4BSkY36mwxYGCK","soc_score":4.5794215,"soc_type":"_doc","soc_timestamp":"2022-01-05 04:59:37.301 +00:00","soc_source":"securityonion:so-ids-2022.01.05"}
`

**Comment by TOoSmOotH** [comment]
It should be listed in the context menu if you are clicking on a specific alert. It does not work for a group of alerts. Can you provide a screenshot?

**Comment by jackface0** [comment, marked as answer]
From the alerts interface I've done a left click "include" on the event and then clicked the ">" arrow to expand.
![image](https://user-images.githubusercontent.com/89797177/148938842-2aaa6c6d-53c2-4e5e-b4ac-30002e2414e7.png)

I've also tried
![image](https://user-images.githubusercontent.com/89797177/148939091-a185750b-6436-45c2-bd33-486c1992d69d.png)

I see this error
![image](https://user-images.githubusercontent.com/89797177/148939199-1ef57395-7a64-4fb8-8622-7a677974de35.png)

I'm sure now this event is normal for the firewall (its trying to do user ID) but I would still like to get the PCAPs working.

**Reply by dougburks** [reply]
The most likely scenario is that the disk is reaching 90% capacity and full packet capture is being purged to prevent the disk from filling.

What is the output of the following commands?
```
df -h
```
```
sudo du --max-depth=1 -h /nsm
```

**Reply by jackface0** [reply]
Ok that makes sense. I have another issue which could be the cause. I had thought I set my HOME_NET variable up correctly during the install but I have seen some Surricata events for legitimate traffic which are HOME_NET -> HOME_NET so maybe I made a mistake. I need to investigate that is configured correctly. I can't get onto the CLI right now but I will check the "du" commands the next chance I get.

**Reply by dougburks** [reply]
Suricata HOME_NET most likely wouldn't cause this issue. In most cases, when users ask about Suricata alerts for HOME_NET -> HOME_NET, the Suricata rules are written for HOME_NET -> EXTERNAL_NET but EXTERNAL_NET is set to ANY (which includes HOME_NET).

**Reply by jackface0** [reply]
You are correct the /nsm directory has filled up. Should I simply add more space or what is the best way forward ? 

df -h output

Filesystem               Size  Used Avail Use% Mounted on
devtmpfs                  16G     0   16G   0% /dev
tmpfs                     16G  300K   16G   1% /dev/shm
tmpfs                     16G   35M   16G   1% /run
tmpfs                     16G     0   16G   0% /sys/fs/cgroup
/dev/mapper/system-root   98G   37G   61G  38% /
/dev/sda2                497M  192M  305M  39% /boot
/dev/mapper/system-nsm   192G  171G   22G  89% /nsm
/dev/mapper/system-tmp   2.0G   53M  1.9G   3% /tmp
overlay                   98G   37G   61G  38% /var/lib/docker/overlay2/aa9a6ccdfdfa5637501d2c6cae711355a10d9c5c06f70e10462993a4a1828bdd/merged
overlay                   98G   37G   61G  38% /var/lib/docker/overlay2/6526d2576d8fe1bb5cf9136ce94abf787658e536604a86ff4035546b58ee27a7/merged
overlay                   98G   37G   61G  38% /var/lib/docker/overlay2/522af97c75577077a76dabed51abf4453a4cb3fbc4027e90d729f50e89f03c65/merged
overlay                   98G   37G   61G  38% /var/lib/docker/overlay2/9f521fe885a686e936633a74b858348c0413498b88b918f7ff6ee3219aa214d7/merged
overlay                   98G   37G   61G  38% /var/lib/docker/overlay2/82eb4b85ca92d49e157edb216db88c6727118f11581172798969fb826c8cc8b6/merged
overlay                   98G   37G   61G  38% /var/lib/docker/overlay2/f2d4a83a8118d9642a25dff227f463c0b5b77d0518d9115e7cfc98ba302e40a4/merged
overlay                   98G   37G   61G  38% /var/lib/docker/overlay2/3786bc74425da3a8c1250bb4da29ce527e8ea166a3cf032061fd909474e3cc36/merged
tmpfs                    3.2G     0  3.2G   0% /run/user/1000
overlay                   98G   37G   61G  38% /var/lib/docker/overlay2/472d277374da82ad09ab535c9b77a2cdc9de83621b0e6102dbfbf94f7fabe5a4/merged
overlay                   98G   37G   61G  38% /var/lib/docker/overlay2/e5fbc8a4559d14c178b30ac55a9531371b4ef32f8a6bc33968b64348f6d61d1a/merged
overlay                   98G   37G   61G  38% /var/lib/docker/overlay2/2a0b386d75e18607d8adbcb929f612ea87c8cca23156b900764d9e93e754032b/merged
overlay                   98G   37G   61G  38% /var/lib/docker/overlay2/dd4c5c57a43f79d2040f66a31f6ea585281b7dcd39400ef54f20834e98c29f8c/merged
overlay                   98G   37G   61G  38% /var/lib/docker/overlay2/66c99c2fae31113c70d2558f348e3c2004d2d7a4bdfbd42ebb877c6e6b3ee81b/merged
overlay                   98G   37G   61G  38% /var/lib/docker/overlay2/2ac73227fdfa74d553d5fee9bccf71c3d27ce4d5c71d9e7519b5fac55ca1c11e/merged
overlay                   98G   37G   61G  38% /var/lib/docker/overlay2/f65f339177ae23a8da59f3275857bf5fa804fe6de9dfc0a8de1a6d1b0f646f61/merged
overlay                   98G   37G   61G  38% /var/lib/docker/overlay2/87ddd3cf921a833cd11287c5537cc0125efc87549eb5d73e8904ae1960485260/merged
overlay                   98G   37G   61G  38% /var/lib/docker/overlay2/739aec1cef868017b6a845fe754da6d11ea8218574c90133aceecde53acb2c8f/merged
overlay                   98G   37G   61G  38% /var/lib/docker/overlay2/9be03b542e653227353e6f8d263e091efff0143c5f0c65a8412f9ccb2a364719/merged
overlay                   98G   37G   61G  38% /var/lib/docker/overlay2/7ae09f7758c5625b5e64a6d018290ae9c0cb9f6380898a7fda98d64a1571ba37/merged
overlay                   98G   37G   61G  38% /var/lib/docker/overlay2/90b6b27ffc570ab3abcce752e3b7a18b58738c58ea448896f10fada267e3cca1/merged
overlay                   98G   37G   61G  38% /var/lib/docker/overlay2/58fc1bc53718ea421902a80deae66dd4f0b6bc7c8eacbb641e4e0a61a6dc2dcd/merged
overlay                   98G   37G   61G  38% /var/lib/docker/overlay2/a1f60fa5c65d6dff395b19d03ef4c0a2d56def9d898ec25c26fdd43e0946d567/merged
overlay                   98G   37G   61G  38% /var/lib/docker/overlay2/dc8fc684c59d5e33c6d4c4060aef4741760f3787a20d12af03b5f69c1d1d3baf/merged
overlay                   98G   37G   61G  38% /var/lib/docker/overlay2/4e63b96f22b79b86fdfef4d2a8952ae383a1330289216bfc5ba97f84072d82a6/merged
overlay                   98G   37G   61G  38% /var/lib/docker/overlay2/d70b15d3830a7850ade60b747a52a7eb029578003faf0e70bf7a4a1a4a432688/merged
overlay                   98G   37G   61G  38% /var/lib/docker/overlay2/2f9355a622c85a49ac13efe939d353bb64e372732db893f333d742b89b469ba2/merged
overlay                   98G   37G   61G  38% /var/lib/docker/overlay2/3e48f11ae4fcaa55da0ad8cb3f8e3e223bb6d4d00a27ce9d982ffed3f47d36e4/merged
overlay                   98G   37G   61G  38% /var/lib/docker/overlay2/17cbd486e26152f4e48c0635676e1a1b40c303e6835c0f9d15586a0d33ea7d17/merged
overlay                   98G   37G   61G  38% /var/lib/docker/overlay2/30f7eb41dd5c0a0e3e60723a21e5abd31288362557734c2c7025208d9a959c2c/merged
overlay                   98G   37G   61G  38% /var/lib/docker/overlay2/e0d43dbdb55320614caf8bf206e512fe8cfda2b3899ec2e6d490af1f3a997cc3/merged
overlay                   98G   37G   61G  38% /var/lib/docker/overlay2/6ed94be9bf813addba0c7c4cc330b3395a522035a6f1482d2151108adc77f7f7/merged
overlay                   98G   37G   61G  38% /var/lib/docker/overlay2/47558c0faa686d8c13100ef0da9dcd34dd953f387b58eb86af4512e4502b879e/merged
overlay                   98G   37G   61G  38% /var/lib/docker/overlay2/5e70878bc0cf05141fd3ab703c55bab1f7054741071be35bf79a5dca4ace0cc9/merged
overlay                   98G   37G   61G  38% /var/lib/docker/overlay2/6a607f78fe1c519cfb893112c78b216e9e257c6b541bf478f2855e689e5cae53/merged
overlay                   98G   37G   61G  38% /var/lib/docker/overlay2/c51ff7a9fdbdb4961cff3e1c9500080b24cd29d5ddf22c56bb46b212df14ee3f/merged

sudo du --max-depth=1 -h /nsm output

13G     /nsm/docker-registry
715M    /nsm/repo
615M    /nsm/backup
35G     /nsm/elasticsearch
63G     /nsm/pcap
4.0K    /nsm/pcaptmp
0       /nsm/pcapout
1.6G    /nsm/pcapindex
2.0G    /nsm/suricata
47G     /nsm/zeek
8.0K    /nsm/soc
238M    /nsm/mysql
1.6G    /nsm/influxdb
1.2M    /nsm/grafana
24K     /nsm/osquery
220M    /nsm/wazuh
55M     /nsm/thehive
2.7G    /nsm/strelka
0       /nsm/import
4.0K    /nsm/logstash
166G    /nsm

For the HOME_NET variable, I had a read of the docs. In my opt/so/saltstack/local/pillar/global.sls I can see the HOME_NET is configured correctly
global:
  soversion: 2.3.90
  hnmanager: '10.4.9.0/24,10.4.10.0/24'
  dockernet: '172.17.0.0'
  mdengine: 'ZEEK'
  ids: 'Suricata'

However, I don't have a section like below so looks like I don't have EXTERNAL_NET configured as !$HOME_NET.
suricata:
  config:
    vars:
      address-groups:
        EXTERNAL_NET: "!$HOME_NET"


I also checked the .sls file in opt/so/saltstack/local/pillar/minions/ but didn't find any "suricata:" section there either. I'll follow the docs and add it into the global.sls. Can I just acknowledge all events and start fresh with my HOME_NET + EXTERNAL_NET variables now configured correctly or is there a better way to clear out all the old events.

**Reply by dougburks** [reply]
> You are correct the /nsm directory has filled up. Should I simply add more space or what is the best way forward ?

Yes, you will most likely want much more than 192G for a real production deployment if you want to store full packet capture for any decent amount of time:
https://docs.securityonion.net/en/2.3/hardware.html#id1

**Reply by jackface0** [reply]
Ok thank you so much for your time and help. I'm just trying to do a POC at the moment so I will look into reducing the data coming in on the span port.
